eppo_client/client.py

```diff
@@ -31,9 +34,86 @@ def __init__(
         )
         self.__poller.start()
 
+    def get_string_assignment(
+        self, subject_key: str, flag_key: str, subject_attributes=dict()
+    ) -> Optional[str]:
+        assigned_variation = self.get_assignment_variation(
+            subject_key, flag_key, subject_attributes, VariationType.STRING
+        )
+        return (
+            assigned_variation.typedValue
+            if assigned_variation is not None
+            else assigned_variation
+        )
+
+    def get_numeric_assignment(
+        self, subject_key: str, flag_key: str, subject_attributes=dict()
+    ) -> Optional[Number]:
+        assigned_variation = self.get_assignment_variation(
+            subject_key, flag_key, subject_attributes, VariationType.NUMERIC
+        )
+        return (
+            assigned_variation.typedValue
+            if assigned_variation is not None
+            else assigned_variation
+        )
+
+    def get_boolean_assignment(
+        self, subject_key: str, flag_key: str, subject_attributes=dict()
+    ) -> Optional[bool]:
+        assigned_variation = self.get_assignment_variation(
+            subject_key, flag_key, subject_attributes, VariationType.BOOLEAN
+        )
+        return (
+            assigned_variation.typedValue
+            if assigned_variation is not None
+            else assigned_variation
+        )
+
+    def get_parsed_json_assignment(
+        self, subject_key: str, flag_key: str, subject_attributes=dict()
+    ) -> Optional[Dict[Any, Any]]:
+        assigned_variation = self.get_assignment_variation(
+            subject_key, flag_key, subject_attributes, VariationType.JSON
+        )
+        return (
+            assigned_variation.typedValue
+            if assigned_variation is not None
+            else assigned_variation
+        )
+
+    def get_json_string_assignment(
+        self, subject_key: str, flag_key: str, subject_attributes=dict()
+    ) -> Optional[str]:
+        assigned_variation = self.get_assignment_variation(
+            subject_key, flag_key, subject_attributes, VariationType.JSON
+        )
+        return (
+            assigned_variation.value
```

Comment:
This is one way to do it--and fine for the foreseeable future. This is what we did in the swift SDK too.

The alternative I did was to call `get_parsed_json_assignment()` and then encode that to JSON. I did that as it removed relying on the "untyped" `value`, and also then the unit test covered both functions. So just an option throwing your way, feel free to use or ignore as you see fit!